### PR TITLE
Added an input for a release id and updated bash script to handle a s…

### PR DESCRIPTION
**Pull request by 09wattry**
…econd argument for the release id

### CODE REVIEW

File: upload-script [hunk 1]
```diff
@@ -10,12 +10,11 @@ if [[ -z "$GITHUB_TOKEN" ]]; then
 fi
 
 # Ensure that there is a pattern specified.
-if [[ -z "$1" ]]; then
+if [[ -n "$2" ]]; then
```
Comment (skx) [inline upload-script:13]: This change I don't understand.

With your PR we expect either:

* One argument == pattern.
* Two argumetns == pattern, release ID

So testing for empty $1 is still the right thing to do for exiting if a pattern is missing.

Comment (09wattry) [reply upload-script:13]: I really do apologize, I added a check for my argument and then removed it. I must have copied it over the original by mistake. I'll update this.

Comment (09wattry) [reply upload-script]: @skx  I amended my previous commit.

**Comment by skx** [comment]
Thanks for updating the argument parsing.

However I've realized there is a problem with positional arguments as I've used this script on a lot of projects.  e.g. I've got it here:

* [skx/puppet-summary](https://github.com/skx/puppet-summary/blob/9630fb70e987fc113e61be3f1033e168fbd8295b/.github/workflows/release.yml#L17)

There you'll see I use a glob `puppet-*` and that means the script gets invoked as:

     $ script puppet-summary-linux-amd64 puppet-summary-linux-i386  ...

That means we _must_ use getopt.  So we'll have a script accept/not the release like so:

     $ script -release=foo arg1
     $ script arg1 arg2 

If you could make a change to use `getopt` then I'll merge.  If not I'll merge and fixup tomorrow rather than leaving this open for too long.

Sorry for only just realizing this, and thank-you once again for the update :)

**Comment by skx** [comment]
I'm an idiot - we can use the environmental variables.

* https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#inputs

I'll fixup.

**Comment by skx** [comment]
Fixed up in 75ce5546020fc1848da842f40240f9fa03e7a3a8 

Tested here:

* https://github.com/skx/release-test/runs/673340100

Hope this works for you :)

**Comment by 09wattry** [comment]
@skx I'm in the GMT-6 time zone so don't worry I only just woke up. Thanks again!

**Pull request merged**